cpp/src/io/statistics/column_stats.h

```diff
@@ -47,11 +49,29 @@ struct stats_column_desc {
   size_type column_offset;         //! < index of the first element relative to the base memory
   const void *column_data_base;    //!< base ptr to column data
   int32_t ts_scale;  //!< timestamp scale (>0: multiply by scale, <0: divide by -scale)
+
+  column_device_view *leaf_column;    //!< Pointer to leaf column
+  column_device_view *parent_column;  //!< Pointer to parent column. Is nullptr if not list type.
 };
 
 struct string_stats {
```

Comment:
Can we use `string_view` instead of `string_stats`? I don't see how they are different.

Reply:
Unfortunately we cannot use `string_view` because they cannot reside inside a union. This is because `string_view` class does not have a trivial constructor.